python/samples/samplebase.py

```diff
@@ -18,6 +18,10 @@ def __init__(self, *args, **kwargs):
         self.parser.add_argument("-p", "--pwmbits", action="store", help="Bits used for PWM. Something between 1..11. Default: 11", default=11, type=int)
         self.parser.add_argument("-l", "--luminance", action="store_true", help="Don't do luminance correction (CIE1931)")
         self.parser.add_argument("-b", "--brightness", action="store", help="Sets brightness level. Default: 100. Range: 1..100", default=100, type=int)
+        self.parser.add_argument("-m", "--hardware_mapping", help="Hardware Mapping: regular, adafruit-hat, adafruit-hat-pwm" , choices=['regular', 'adafruit-hat', 'adafruit-hat-pwm'], type=str)
```

Comment:
(as suggested in my comment above, we should move these options probably into the implementation of RGBMatrixOptions, then this struct deals with the properties and its own flags, so everything is self-contained. Then some call to options.InitFromFlags(self.parser) or something.)